src/controllers/list.js

```diff
@@ -346,9 +359,9 @@ define(['globalize', 'listView', 'layoutManager', 'userSettings', 'focusManager'
     }
 
     function showFilterMenu() {
-        var instance = this;
+        const instance = this;
 
-        require(['filterMenu'], function (FilterMenu) {
+        import('filterMenu').then(({default: FilterMenu}) => {
             new FilterMenu().show({
```

Comment:
I'm going to bring these file changes into https://github.com/jellyfin/jellyfin-web/pull/1728, as I needed to migrate the `list.js` controller as well to get things working for the `filtermenu`. I'm just going to make a singular change from your edits here and migrate `.show()` to be called as a static method, since it doesn't act like an instance method.

Reply:
Also, `AlphaNumericShortcuts.default({})` and `AlphaPicker.default({})` can be migrated to using their constructors directly now. `AlphaNumericShortcuts({})` & `AlphaPicker({})`

Reply:
Cool, that sounds good, I've removed the .defaults as well.